pkg/api/registry/registry.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // versions:
-// 	protoc-gen-go v1.26.0
-// 	protoc        v3.14.0
```

Comment:
Please use these versions of generating tools to be consistent with other models.
Updating for new versions should be done in separate PR.

Reply:
done